en: update docs for getting started (#117)
pingcap/docs-tidb-operator: en/deploy-tidb-from-kubernetes-gke.md

If you see `Ready` for all nodes, congratulations! You've setup your first Kubernetes cluster.

## Install Helm

[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.11.0 and < 2.16.4. The installation steps are as follows:

> 1. Refer to [Helm official documentation](https://v2.helm.sh/docs/using_helm/#installing-helm) to install Helm client.

Reviewer: ```suggestion
1. Refer to [Helm official documentation](https://v2.helm.sh/docs/using_helm/#installing-helm) to install the Helm client.
```

> 2. Install Helm server.

Reviewer: ```suggestion
2. Install the Helm server.
```

    Apply the `RBAC` rule required by the `tiller` component in the cluster and install `tiller`:

    {{< copyable "shell-regular" >}}

    ```shell
    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/tiller-rbac.yaml && \
    helm init --service-account=tiller --upgrade
    ```

    To confirm that the `tiller` Pod is in the `running` state, run the following command:

    ```shell
    kubectl get po -n kube-system -l name=tiller
    ```

3. Add the repository:

    {{< copyable "shell-regular" >}}

    ```shell
    helm repo add pingcap https://charts.pingcap.org/
    ```

    Use `helm search` to search the chart provided by PingCAP:

    {{< copyable "shell-regular" >}}

    ```shell
    helm search pingcap -l
    ```

## Deploy TiDB Operator

> TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` custom resource type.

Reviewer: ```suggestion
TiDB Operator uses [CRD (Custom Resource Definition)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to extend Kubernetes. Therefore, to use TiDB Operator, you must first create the `TidbCluster` CRD.
```

{{< copyable "shell-regular" >}}

```shell
kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && \
kubectl get crd tidbclusters.pingcap.com
```

> After `TidbCluster` custom resource type is created, install TiDB Operator in your Kubernetes cluster.

Reviewer: ```suggestion
After `TidbCluster` CRD is created, install TiDB Operator in your Kubernetes cluster.
```

    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/persistent-disk.yaml
    ```

## Deploy the TiDB cluster

> Deploy the TiDB cluster by the following steps:

Reviewer: ```suggestion
To deploy the TiDB cluster, perform the following steps:
```

1. Create `Namespace`:

    {{< copyable "shell-regular" >}}

    ```shell
    kubectl create namespace demo
    ```

2. Deploy the TiDB cluster:

    {{< copyable "shell-regular" >}}

    ``` shell
    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/basic/tidb-cluster.yaml -n demo
    ```

3. Deploy the TiDB cluster monitor:

    {{< copyable "shell-regular" >}}

    ``` shell
    kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/basic/tidb-monitor.yaml -n demo
    ```

4. View the Pod status:

    {{< copyable "shell-regular" >}}

    ``` shell
    kubectl get po -n demo
    ```

## Connect to the TiDB cluster

> There can be a small delay between the pod being up and running, and the service being available. You can view the service status with the following command:

Reviewer: ```suggestion
There can be a small delay between the pod being up and running, and the service being available. You can view the service status using the following command:
```
